syntax = "proto3";

package crocks.pb;

message NodeInfo {
  string address = 1;
  repeated int32 shards = 2;
}

message ClusterInfo {
  int32 num_shards = 1;
  repeated NodeInfo nodes = 3;
}
